CHANGELOG.md

```diff
@@ -39,7 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Fix working with `embedding_sizes` in `202-NN_examples` notebook ([#489](https://github.com/etna-team/etna/pull/489))
 - Disallow dropping target in `TSDataset.drop_features` ([#491](https://github.com/etna-team/etna/pull/491))
-- 
+- Optimize memory usage in `TFTNativeModel` ([#494](https://github.com/etna-team/etna/pull/494))
```

Comment:
We could write it more specific:
```
Optimize memory usage in `TFTNativeModel` by eliminating copying during making samples
```

We could also update issue name and PR name in a similar way.